kvrocks.conf

```diff
@@ -578,7 +578,7 @@ rocksdb.min_blob_size 4096
 # opened once this limit is reached.
 #
 # Default: 128 M
-rocksdb.blob_file_size 128
+rocksdb.blob_file_size 134217728
```

Comment:
I think we can submit another PR to resolve this issue? also need to change the default value in config.cc(`min_blob_size` was the same problem).